.gitignore

```diff
@@ -14,4 +14,5 @@
 gke-policy
 
 terraform/.terraform*
-terraform/terraform*
\ No newline at end of file
+terraform/terraform.tfvars
```

Comment:
we could add here also:
terraform/*.tfstate
terraform/*.tfstate.*

Reply:
Added, thanks.